medvis/dicom/src/io/gdcmvolumereader.cpp

```diff
@@ -509,30 +509,30 @@ std::shared_ptr<VolumeSequence> GdcmVolumeReader::tryReadDICOMDIR(
 /**
  * Entry point of the reader, called from VolumeSource processor
  */
-std::shared_ptr<VolumeSequence> GdcmVolumeReader::readData(const std::string_view filePath) {
+std::shared_ptr<VolumeSequence> GdcmVolumeReader::readData(const std::filesystem::path& filePath) {
     auto path = filePath;
-    if (!filesystem::fileExists(path)) {
-        std::string newPath = filesystem::addBasePath(path);
-        if (filesystem::fileExists(newPath)) {
+    if (!std::filesystem::exists(path)) {
```

Comment:
I would probably use std::filesystem::is_regular_file here to make use it is a file... exists will return true for lots of things...